HyperPlay-Gaming/hyperplay-desktop-client
# [FIX] RunWineCommand not working for Non-HyperPlay Games

Found this issue while testing the MarketWars on macOS but it affects and GOG games as well.
Commands like `winecfg` and `winetricks` doesn't work because it uses the `ìsNative` from HyperPlay instead of the one from `sideloaded`.

### How to test

- On mac or linux.
- Opens any Windows game settings and click the 'WineCFG' button
- On `main` it should not open the wine configuration window.
- On this branch it should appear.

---

Use the following Checklist if you have changed something on the Backend or Frontend:
